oddbird/oddcontrast
# Group formats

![](https://source.unsplash.com/featured/?cute,animal&group)

## Description
Groups the formats by groups/space, and moves the out of gamut warning to the group rather than the individual format.

https://www.figma.com/file/JtQTGIpwoaOcqcWfsoeOk1/Browser-bugs-%26-other-warnings?type=design&node-id=5-146&mode=design&t=c0xZQdteuNYmK61O-0

## Steps to test/reproduce

https://deploy-preview-137--oddcontrast.netlify.app/

1. With a format that isn't p3 selected, see that there are 3 groups, and the selected format is missing from its group
2. With p3 selected, see that there are only 2 groups.
3. Test out of gamut colors, most easily with oklch, and see that the messages show with the group name. 
4. Check that the links go to the Color 4 [Out of Gamut](https://www.w3.org/TR/css-color-4/#out-of-gamut) explainer (is that the right place?)


## Show me
(Styling is still needed)
<img width="1030" alt="image" src="https://github.com/oddbird/oddcontrast/assets/167908/6e9fe5f7-5dd8-492b-972d-5245406c7c5e">

Related to #95
